pkg/client/connector/k8s_config.go

```diff
@@ -1,17 +1,50 @@
 package connector
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"sort"
 
 	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/rest"
 
 	"github.com/datawire/ambassador/pkg/kates"
+	"github.com/telepresenceio/telepresence/v2/pkg/iputil"
 )
 
+// The dns config part of the kubeconfigExtension that is read from the kubeconfig
+type dns struct {
```

Comment:
Again, unclear name. Perhaps `dnsConfig`?

Reply:
Actually, `dnsConfig` is almost just as bad.  How about `kubeconfigDNS`?

Reply:
This is a subtype in the`kubeconfigExtension`. I think `kubconfigDNS` in `kubeconfigExtension` is a bit of an overkill. Hence just `dns`.